packages/clarity-js/src/core/config.ts

```diff
@@ -20,7 +20,8 @@ let config: Config = {
     action: null,
     dob: null,
     delayDom: false,
-    throttleDom: true
+    throttleDom: true,
+    conversions: false
```

Comment:
Nit: I like adding a comma at the end as it will cause less merge conflicts.

Reply:
fair :)